ERCS/erc-7518.md

```diff
@@ -0,0 +1,606 @@
+  */
+  function canTransfer(address from, address to, uint id, uint amount, bytes calldata data) external view returns (bool status);
+
+  /**
+  * @dev lock token till a particular block time.
+  * @param account The address of the account for which tokens will be locked.
+  * @param id The token ID.
+  * @param amount The amount of tokens to be locked for the account.
+  * @param releaseTime The timestamp indicating when the locked tokens can be released.
+  * @return bool Returns true if the tokens are successfully locked, otherwise false.
+  */
+  function lockTokens(address account, uint id, uint256 amount, uint256 releaseTime) external returns (bool);
+
+  /**
+  * @dev Unlocks tokens that have crossed the release time for a specific account and id.
+  * @param account The address of the account to unlock tokens for.
+  * @param id The token ID.
+  */
+  function unlockToken(address account, uint256 id) external;
+
+  /**
+  * @dev Force transfer in cases like recovery of tokens.
+  * @param from The address to transfer tokens from.
+  * @param to The address to transfer tokens to.
+  * @param id The token ID.
+  * @param amount The amount of tokens to transfer.
+  * @param data Additional data related to the transfer.
+  * @return A boolean value indicating whether the operation was successful.
+  */
+  function forceTransfer(address from, address to, uint256 id, uint256 amount, bytes memory data) external returns (bool);
+
+  /**
+  * @dev Freezes specified address.
+  * @param account The address to be frozen.
+  * @param data Additional data related to the freeze operation.
+  * @return A boolean value indicating whether the operation was successful.
+  */
+  function freezeAddress(address account, bytes calldata data) external returns (bool);
+
+  /**
+  * @dev Unfreezes specified address.
+  * @param account The address to be unfrozen.
+  * @param data Additional data related to the unfreeze operation.
+  * @return A boolean value indicating whether the operation was successful.
+  */
+  function unFreeze(address account, bytes memory data) external returns (bool);
+
+
+  /**
+  * @dev Sends payout to single address with corresponding amounts.
+  * @param to address to send the payouts to.
+  * @param amount amount representing the payouts to be sent.
+  * @return A boolean indicating whether the batch payouts were successful.
+  */* 
+  function payout(address calldata to, uint256 calldata amount) public returns (bool);
+
+  /**
+  * @dev Sends batch payouts to multiple addresses with corresponding amounts.
+  * @param to An array of addresses to send the payouts to.
+  * @param amount An array of amounts representing the payouts to be sent.
+  * @return A boolean indicating whether the batch payouts were successful.
+  */
+  function batchPayout(address[] calldata to, uint256[] calldata amount) public returns (bool);
+}
+```
+
+### Methods for token
+
+#### transferableBalance
```

Comment:
Please put code snippets in backticks (\`):

```suggestion
#### `transferableBalance`
```

This applies to the rest of the document as well.

---

Note that if you tick the "Allow Edits from Maintainers" box on the pull request, I can make minor changes like this for you.

Reply:
'Allow Edits from Maintainers' option is disabled for organization repositories. However, I have added you as a collaborator to the [zoniqx:eip-7518](https://github.com/zoniqx/ERCs/tree/eip-7518) branch.